### PR TITLE
Add version information to most wiki feature and framework pages

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Title
- Add "removed" style of version to front matter
- Update and improve documentation guidelines and tips
- Add a note to `sitting` feature and framework pages about move to ACEX
- Add `description` to all wiki pages and fix capitalization/closing point on others

### CODE REVIEW

File: docs/_layouts/wiki.html [hunk 1]
```diff
@@ -9,28 +9,36 @@
     <div class="row">
         <div class="medium-8 small-12 columns">
             <span>
+            {% if page.version.removed %}
+                {% assign type = 'removed' %}
+                {% assign typeCapital = 'Removed' %}
+            {% else %}
+                {% assign type = 'added' %}
+                {% assign typeCapital = 'Added' %}
```
Comment (Glowbal) [inline docs/_layouts/wiki.html]: What about `introduced` instead of `added`?

Comment (jonpas) [inline docs/_layouts/wiki.html]: I like `added` more as it's more direct and in-line with our changelogs.

File: docs/wiki/development/documentation-guidelines-and-tips.md [hunk 2]
```diff
@@ -6,59 +6,96 @@ parent: wiki
...
 ```
 ---
 layout: wiki
-title: Mission Modules
+title: Component
+description: Description of the feature.
 group: feature
+category: interaction
 parent: wiki
+version:
```
Comment (bux578) [inline docs/wiki/development/documentation-guidelines-and-tips.md]: misses `mod: ace(x)`

Comment (jonpas) [inline docs/wiki/development/documentation-guidelines-and-tips.md]: But, what is the point of having `mod` in development pages?

**Pull request merged**